packages/dm-core/src/components/MediaContent.tsx

```diff
@@ -69,24 +76,44 @@ export const MediaContent = (props: MediaContentProps): ReactElement => {
           }}
         />
       )
-    } else {
+    } else if (applicationFiletypes.includes(filetype)) {
       return (
-        <iframe
+        <embed
           title={meta.title}
           src={blobUrl}
-          style={{ width: '100%', height: 'auto' }}
+          type={mime.getType(filetype) || 'application/octet-stream'}
+          style={{ width: '100%', height: '100%' }}
           height={config.height}
           width={config.width}
-          role='document'
+          data-testid='embeded-document'
         />
       )
+    } else {
+      return (
+        <div
+          className='border border-equinor-green bg-equinor-lightgreen p-3 '
+          data-testid='unknown-file-message'
+        >
+          <h2 className='text-lg text-equinor-green font-medium'>
+            Unknown filetype
+          </h2>
+          <p className='flex gap-1 mb-3'>
+            The filetype <code>{meta.filetype}</code> is not known, and can not
+            be previewed. You may still download the file
+          </p>
+          <Button download={`${meta.title}.${meta.filetype}`} href={blobUrl}>
+            <Icon size={16} data={download} />
+            Download
+          </Button>
```

Comment:
I would change the wording here a bit.
The filetype is not "unknown", we just don't have a compatible viewer for it.
So something in the lines of;
"No preview available. Download the file to open it with another application."